src/lib/modernExtend.ts

```diff
@@ -358,17 +378,23 @@ export function numeric(args: NumericArgs): ModernExtend {
         },
     }];
 
-    return {exposes: [expose], fromZigbee, toZigbee, isModernExtend: true};
+    const configure: Configure = async (device, coordinatorEndpoint, logger) => {
```

Comment:
Please also add the same `configure` to `export function binary` (so `binary`, `numeric` and `enumLookup` have the same, I guess it would be good to create a function instead to prevent code duplication.